packages/editor/src/components/parts/permissions/PermissionsPart.tsx

```diff
@@ -0,0 +1,28 @@
+import { PartProps, usePartDirty, usePartState } from '../../../components/editors';
+import { PathContext, useValidations } from '../../../context';
+import { Checkbox } from '../../../components/widgets';
+import { usePermissionsData } from './usePermissionsData';
+import { PermissionsData } from '@axonivy/inscription-protocol';
+
+export function usePermissionsPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = usePermissionsData();
+  const compareData = (data: PermissionsData) => [data];
+  const validation = useValidations(['permissions']);
+  const state = usePartState(compareData(defaultConfig), compareData(config), validation);
+  const dirty = usePartDirty(compareData(initConfig), compareData(config));
+  return { name: 'Permissions', state, reset: { dirty, action: () => reset() }, content: <PermissionsPart /> };
+}
+
+const PermissionsPart = () => {
+  const { config, update } = usePermissionsData();
+
+  return (
+    <PathContext path='permissions'>

```

Comment:
You don't need a path context here, as you have no validation messages nor a script context